static/js/admin.js

```diff
@@ -759,13 +765,20 @@ function _setup_page() {
                 full_name: JSON.stringify(full_name.val()),
             };
 
+            if (owner.val() !== undefined && owner.val() !== "") {
+                url = "/json/bots/" + email;
```

Comment:
Should we be using `encodeURIComponent` here?  Also, what happens here if both `full_name` and `owner` get set when you hit the submit button for the form?